index.rst

```diff
@@ -6,65 +6,110 @@
 Abstract
 ========
 
-In order to calibrate the AuxTel instruments, particularly the LATISS spectrograph, we want to illuminate the focal plane with monochromatic light at a range of wavelengths. This tech note describes the Rubin Observatory Auxilliary Telescope Calibration Illumination System, its basic operation, and a test dataset.
+In order to calibrate the AuxTel instruments, particularly the LATISS spectrograph, we want to illuminate the focal plane with monochromatic light at a range of wavelengths. This tech note describes the Rubin Observatory Auxilliary Telescope Calibration Illumination System, its basic operation, maintenance and trouble-shooting, and a test dataset.
 
 Overview
-================
+========
 The Auxilliary Telescope (AuxTel) Calibration Illumination System will be used to illuminate the white calibration screen in the AuxTel with a tunable light source, ranging in wavelength from 350-1200 nm. The illumination system will sit on the dome floor directly below the calibration screen. 
 
-The system was developed and built in Tucson. This document is meant to give an overview of the system and document the assembly and integration process of the system when shipped to Chile. This document also includes an overview of how the system should be operated with sample code, and links to all documentation related to this system so it can be more readily accessed. 
-
 The work to prepare this subsystem for shipment was tracked in the JIRA Epic `SITCOM-259c <https://jira.lsstcorp.org/browse/SITCOM-259>`__.
 
 Some documentation on this system can be accessed on `Docushare Collection-11943 <https://docushare.lsst.org/docushare/dsweb/View/Collection-11943>`__.
 
 There are additional details about control of the LabJack in `Tech Note 36 <https://tstn-036.lsst.io>`__.
 An overview of the data is in a Jupyter Notebook in the github for this tech note. 
 
-
-Hardware
-==============
+System Description
+==================
 There are five main functional elements of the AuxTel Illumination System:
 
-1. **The White Light Source** supplies the light, and requires a **chiller** to remove heat from the beam.
+1. **The White Light Source (WLS)** supplies the light, and requires a **chiller** to remove heat from the beam.
```

Comment:
"remove heat from the beam" is technically correct, but the important thing is does is remove mid-infrared light (heat) which would otherwise damage components inside the monochromator.

Reply:
Ok, so you explain this down below, so consider modifying this to something like:
"...and requires a **chiller** to remove undesired wavelengths (i.e. heat) from the beam.

Reply:
done